packages/back-end/src/api/features/postFeature.ts

```diff
@@ -42,19 +42,22 @@ export const parseJsonSchemaForEnterprise = (
   org: OrganizationInterface,
   jsonSchema: string | undefined
 ) => {
-  const defaultJsonSchema = {
+  const jsonSchemaWrapper = {
     schema: "",
     date: new Date(),
-    enabled: false,
+    enabled: true,
```

Comment:
Shouldn't this default to false and only get set to true if a schema is supplied and the org is enterprise?

Reply:
That was a silly mistake. Fixed